README.md

```diff
@@ -127,27 +121,47 @@ paravance:
 ```
 
 2) Create an new inventory file in the `inventories` subdirectory
-(copy paste the sample inventory) and change the group of the conductor service :
+(copy paste the sample inventory) and change the group of the
+conductor service:
+
+```
+[nova-conductor:children]
+conductor-node
+```
+
+3) In the configuration file, points the inventory to use to this new
+inventory.
 
+4) Launch the deployment as usual, and you'll get the `nova-conductor`
+on a dedicated node.
 
 ### Configuration tuning
 
-At some point, Kolla default parameters won't fit your needs. Kolla provides a
-mechanism to override custom section of configuration files but isn't applicable
-in our case (at least in the corresponding branch). So we implement a *quick and dirty*
-way of patching Kolla code to enable custom configuration files to be used
-(and by extension custom kolla code).
-See the possible patch declaration in `ansible/group_vars/all.yml`. Patches should be added in the configuration file of the experiment.
+At some point, Kolla default parameters won't fit your needs. Kolla
+provides a mechanism to override custom section of configuration files
+but isn't applicable in our case (at least in the corresponding
+branch). So we implement a *quick and dirty* way of patching Kolla
+code to enable custom configuration files to be used (and by extension
+custom kolla code). See the possible patch declaration in
+`ansible/group_vars/all.yml`. Patches should be added in the
+configuration file of the experiment.
+
+## Contribute
```

Comment:
+1 for this section.
But I think most of it should be placed on the installation part.